app/budget/directives/modals/addExpenseItem/addExpenseItem.html

```diff
@@ -0,0 +1,98 @@
+<div class="add-expense-item">
+	<form role="form" class="form-horizontal add-expense-item__container">
+		<!--Term-->
+		<div class="form-group">
+			<label class="col-sm-4 control-label">Term</label>
+			<div class="col-sm-5">
+				<div class="input-group" class="type-selection-container">
+					<button class="btn btn-type-dropdown dropdown-toggle"
+									type="button" data-toggle="dropdown"
+									class="btn-type-dropdown">
+						<span class="btn-type-dropdown-label">
+							{{ newExpenseItem.termDescription || 'Select a term'}}
+						</span>
+						<span class="caret"></span>
+					</button>
+
+					<ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+						<li ng-repeat="term in state.ui.termNav.budgetScenarioDropdownTerms | orderBy:'description'" class="type-item">
```

Comment:
I feel like it makes more sense not to order it alphabetical here.

Reply:
Yep, makes sense.